test/win-path-tests.js

```diff
@@ -48,7 +48,9 @@ describe('Paths', () => {
         const env = new CompilationEnvironment(compilerProps);
 
         const compiler = new WineCL(info, env);
-        compiler.filename("/tmp/123456/output.s").should.equal("Z:/tmp/123456/output.s");
+        if (process.platform === 'linux' || process.platform === 'darwin') {
```

Comment:
Why is this check needed? So it's not ran on windows? Shouldn't this function's behaviour hold regardless of platform? (Same with L69)

Reply:
The if statement has to be somewhere, better it to be in the tests than having to add exceptions throughout the code